Exercises-4-OOP/src/ExercisesOnClasses/Circle.java

```diff
@@ -0,0 +1,25 @@
+package ExercisesOnClasses;
```

Comment:
```suggestion
package exercisesonclasses;
```
Package name should contain only lower case letters.
The same for other packages.

Reply:
done

Reply:
frtrtrt